src/did/resolver.ts

```diff
@@ -0,0 +1,27 @@
+import { Resolver, DIDDocument } from "did-resolver";
+import { getResolver as ethrGetResolver } from "ethr-did-resolver";
+import { getResolver as webGetResolver } from "web-did-resolver";
+import { INFURA_API_KEY } from "../config";
+
+const providerConfig = {
+  networks: [{ name: "mainnet", rpcUrl: `https://mainnet.infura.io/v3/${INFURA_API_KEY}` }],
```

Comment:
nothing for ropsten / rinkeby ?

Reply:
Nope:
1. Not now, I dont care that it works perfectly for all cases yet until we got the first document and code that we are satisfied with.
2. Even in later stages, they can still use `did:ethr` directly `did:ethr:ropsten:0xE6Fe788d8ca214A080b0f6aC7F48480b2AEfa9a6` looks quite silly.